backend/WebUI/routers.go

```diff
@@ -147,70 +187,80 @@ var routes = Routes{
 	{
 		"GetSubscribers",
 		http.MethodGet,
-		"/subscriber",
+		"/subscribers",
 		GetSubscribers,
 	},
 
 	{
 		"GetSubscriberByID",
 		http.MethodGet,
-		"/subscriber/:ueId/:servingPlmnId",
+		"/subscribers/:supi/:plmnId",
 		GetSubscriberByID,
 	},
 
 	{
 		"PostSubscriberByID",
 		http.MethodPost,
-		"/subscriber/:ueId/:servingPlmnId",
+		"/subscribers/:supi/:plmnId",
 		PostSubscriberByID,
 	},
 
 	{
-		"PostMultiSubscriber",
+		"PostMultipleSubscribersByID",
 		http.MethodPost,
-		"/subscriber/:ueId/:servingPlmnId/:userNumber",
-		PostSubscriberByID,
+		"/subscribers/:supi/:plmnId/:userNumber",
+		PostMultipleSubscribersByID,
 	},
 
 	{
 		"PutSubscriberByID",
 		http.MethodPut,
-		"/subscriber/:ueId/:servingPlmnId",
+		"/subscribers/:supi/:plmnId",
 		PutSubscriberByID,
 	},
 
 	{
 		"DeleteSubscriberByID",
 		http.MethodDelete,
-		"/subscriber/:ueId/:servingPlmnId",
+		"/subscribers/:supi/:plmnId",
 		DeleteSubscriberByID,
 	},
 
 	{
 		"PatchSubscriberByID",
 		http.MethodPatch,
-		"/subscriber/:ueId/:servingPlmnId",
+		"/subscribers/:supi/:plmnId",
 		PatchSubscriberByID,
 	},
 
+	// AMF currently returns all contexts at once
 	{
-		"Registered UE Context",
+		"GetAmfUeContexts",
 		http.MethodGet,
-		"/registered-ue-context",
-		GetRegisteredUEContext,
+		"/amf-ue-contexts",
+		GetAmfUeContexts,
 	},
 
-	{
-		"Individual Registered UE Context",
-		http.MethodGet,
-		"/registered-ue-context/:supi",
-		GetRegisteredUEContext,
-	},
+	// AMF currently returns all contexts at once
+	//{
+	//	"GetAmfUeContextByID",
+	//	http.MethodGet,
+	//	"/amf-ue-contexts/:supi",
+	//	GetAmfUeContextByID,
+	//},
+
+	// not implemented, as not required, only need a single context
```

Comment:
Should we remark this part as TODO item?

Reply:
Actually this is simply not needed, I left the code in because it might be needed in the future